agent/tool-scripts/postprocess/gold/proc-vmstat/csv/proc-vmstat_pgalloc_delta_sec.csv

```diff
@@ -4,8 +4,8 @@ timestamp_ms,dma,dma32,movable,normal
 1424904539496.22,0,2379.14407641464,0,3121.66887987701
 1424904542498.77,0,979.833791995655,0,1874.73977401208
 1424904545501.49,0,1439.361659485,0,1657.16418732007
-1424904548503.95,0,742.724309070785,0,1710.59733246079
-1424904551507.05,0,1058.2397844415,0,1479.13817573604
+1424904548503.96,0,742.721832944019,0,1710.59162959663
+1424904551507.05,0,1058.24331171904,0,1479.14310593328
```

Comment:
If perl can't get two decimal places right, why maintain twelve decimal places? Not sure this is worth worrying about here, but in general, we need to worry about (false) accuracy.

Reply:
Definitely not worth worrying about here, as these metrics will be replaced by PCP and/or Prometheus in the future.

In general, the Perl change appears to have happened at Perl 5 v30.  But it is stable after v30 it seems.